src/NuGet.Services.Staging.Database/Models/StageContext.cs

```diff
@@ -15,6 +16,8 @@ public class StageContext : DbContext
 
         public virtual DbSet<StageMembership> StageMemberships { get; set; }
 
+        public virtual DbSet<PackageMetadata> PackagesMetadata { get; set; } 

```

Comment:
PackageMetadata is the name for this kind of info in all other projects. I think the weirdness Maarten was refereing to in "PackageS Metadata"

---
In reply to: [64719651](https://github.com/NuGet/NuGet.Staging/pull/50#discussion_r64719651) [](ancestors = 64719651)